userspace/engine/logger.cpp

```diff
@@ -128,8 +128,8 @@ void falco_logger::log(falco_logger::level priority, const std::string&& msg) {
 			}
 		} else {
 			const struct tm* ltm = std::localtime(&result);
-			static char format[] = "WWW MMM DD HH:mm:ss YYYY";
-			char tstr[sizeof(format)];
+			static char format[] = "%a %b %d %H:%M:%S %Y";
+			char tstr[sizeof("aaa bbb ddd HH:MM:SS YYYY")];
 			std::strftime(tstr, sizeof(tstr), format, ltm);
```

Comment:
Sorry for that one, I don't know how I managed to mix the format itself and the string that would show it.
I would like to propose the following which is also more in line with modern c++ suggestions.
```suggestion
			char tstr[std::size("WWW MMM DD HH:mm:ss YYYY")];
			std::strftime(std::data(tstr), std::size(tstr), "%a %b %d %H:%M:%S %Y", ltm);
```